CHANGELOG.md

```diff
@@ -7,186 +7,83 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
-- allow removing a component when it is invalid
-- add `getSchema` to extension life cycle
-- validate extension's rawConfig against extension schema
-- changed `--include-unmodified` to `--all`
+- New `bit watch` command for building components upon file modifications.
+- New `bit eject` for removing local components and installing them as packages by an NPM client
+- Support dynamic namespaces (replaced the namespace/name format with a dynamic name that can have multiple slashes to indicate a hierarchical namespace).
+- Support components with binary files (or non-supported extensions) as the only files.
+- Support ids with wildcards (e.g. `bit tag "utils/*"`) for the following commands: `tag`, `untag`, `remove`, `untrack`, `checkout`, `merge`, `diff` and `export`.
+- Support mix syntax of typescript and javascript inside .ts file
+- Added react docs parsing to extract the description of the properties correctly.
+- Support flow types in react doc generation.
+- Support Vue files with typescript.
+- Support configuring Git executable path.
+- Support the new jsx syntax changes by Babel.
+- Support print multiple external (build / test) errors.
+- Support adding the project `package.json` file to a component.
+- Support `import ~` from a local (authored) file to an imported sass component.
+- Add programmatic API for add multiple components.
+- Set the only dist file as main file in package.json (in case there is only one).
+- New programmatic API for add multiple components.
+- Allow removing a component when it is invalid.
 
-## [13.0.5-dev.2 - 2018-08-20]
+### Changes
 
-- change `--conf` on `bit import` to be a path to the config dir
+- Improved performance for traking, versioning and exporting components by up to 700%.
+- CSS parser replaced for better import syntax support.
+- Replace caporal package with commander to support node 6.
+- Major refactor to write components.
+- Better error when a component was tagged without its dependencies.
+- Avoid installing node_modules in sub directories when the package manager allow it to be installed from the root dir.
+- Make bit version command faster and support both `bit -v` and `bit -V` to get bit version.
+- Update tty-table, flow-coverage-report and mocha-appveyor-reporter for security reasons.
+- Improve exception handling for old clients connecting to a newer server.
+- Shorten the generated component ID to the minimum possible.
+- Return status code 1 when bit test has failing tests.
+- Suppress an exception of directory-is-empty when adding multiple components and some of them are empty, show a warning instead.
+- Improve "missing a main file" error when adding multiple components to print the problematic components.
+- Improve performance by caching objects after loading them.
+- Fix ci-update command with component version number.
+- Show a descriptive error when failed to parse css, sass, scss and less files.
```

Comment:
this feature has been removed yesterday (see https://github.com/teambit/bit-javascript/pull/83).